lib/data_update_scripts/20200409065412_add_classified_listing_categories.rb

```diff
@@ -0,0 +1,26 @@
+module DataUpdateScripts
+  class AddClassifiedListingCategories
```

Comment:
This seemed like a good idea when I wrote it and was useful for local testing, but it also alerted me to the fact that we may need to look into these scripts in the context of generalization: they are checked in and one needs to run them for the app to start up, yet some of them may be very DEV specific (like this one would add DEV categories for everyone). I'll turn this into a Rake task tomorrow that we can remove later.

Reply:
I have the same thoughts on generalization. I also think it would be useful to create categories while seeding.

Reply:
Yeah, good idea, I didn't think about how DEV oriented are these categories. Should they be saved in the site config?

Reply:
I thought about SiteConfig at first, but I think having a separate model and an admin interface for it makes more sense.